README.md

```diff
@@ -51,6 +51,7 @@ The [Internet Computer](https://internetcomputer.org/) is a public blockchain th
 
 ### Tutorials and Samples
 
+- [Cross-Chain ETH Payment and E-Commerce](https://github.com/b3hr4d/eth_payment_tutorial) - A step-by-step guide to building a cross-chain payment and e-commerce platform on the Internet Computer using Ethereum.
```

Comment:
Please move this to the advanced section. This first section is in particular for collections of tutorials and samples.

Reply:
I made new [PR](https://github.com/dfinity/awesome-internet-computer/pull/100)